examples/index.html

```diff
@@ -19,7 +19,8 @@
   <body>
     <h1>StackBlitz SDK Examples</h1>
     <ul>
-      <li><a href="/examples/open-embed-project-id/">Open and embed a StackBlitz project</a></li>
+      <li><a href="/examples/open-embed-project-id/">Open and embed a StackBlitz EngineBlock project</a></li>
+      <li><a href="/examples/open-embed-webcontainer/">Open and embed a StackBlitz WebContainer project</a></li>
```

Comment:
Added a new example to test loading WebContainer projects with and without the CORP headers.